torch/csrc/jit/passes/quantization.cpp

```diff
@@ -452,7 +456,7 @@ class QuantizeHelper {
   c10::optional<script::Module> findChildModuleToQuantize(
       Value* child_instance);
   void quantizeTensor(Value* v);
-  void removeObserver(Value* v, const std::string& observer_name);
+  Value* removeObserver(Value* v, const std::string& observer_name);
```

Comment:
can you add some comments to this function? especially input Value and return value?